api-parent/core-api/src/main/java/com/sap/cloud/environment/servicebinding/api/DefaultServiceBinding.java

```diff
@@ -139,7 +139,15 @@ public Map<String, Object> getCredentials()
     @Override
     public int hashCode()
     {
-        return Objects.hash(getName(), getServiceName(), getServicePlan(), getTags(), getCredentials(), properties);
+        return Objects
+            .hash(
+                getName(),
+                getServiceName(),
+                getServiceIdentifier(),
+                getServicePlan(),
+                getTags(),
+                getCredentials(),
+                properties);
```

Comment:
This is a drive-by change since we forgot to include the newly introduced `getServiceIdentifier` into the `equals` and `hashCode` implementation